scripts/integration-tests/e2e-babel-old-version.sh

```diff
@@ -43,10 +43,7 @@ node "$PWD"/scripts/integration-tests/utils/bump-babel-dependencies.js
   fs.writeFileSync('./package.json', JSON.stringify(pkg, null, 2));
 "
 
-rm yarn.lock
```

Comment:
The `e2e-babel-old-version` test fails because the `browserslist` lock is also removed: https://app.circleci.com/pipelines/github/babel/babel/3678/workflows/a3f16b77-88e7-4beb-904c-a7bad43e9591/jobs/27418

Reply:
Can we instead modify package.json pinning only browserslist?

Reply:
We didn't upgrade `browserslist` in this PR so it is implicitly pinned in the lock file. It was pinned as `4.12.0`, same as the version declared in the deleted `packages/babel-compat-data/yarn.lock`. I don't think we need to pin it on package.json except we meant to avoid a security flaw or certain version does not keep backward compatibility.

Reply:
I meant to pin it in only in this test, like we do for `@babel/core`

Reply:
Ah I see, the `e2e-test` here is meant to update all deps. I went ahead and pinned it globally.